### ISSUE TITLE
Error detection in makePackages.sh

### ISSUE THREAD

**Issue by mode89** [ghcjs/ghcjs#805]
Shouldn't this be uncommented?
https://github.com/ghcjs/ghcjs/blob/2453b111a20ac99a1b1d4de2c8aaa824be4114f5/utils/makePackages.sh#L2

**Comment by mode89** [comment]
I'm currently trying to build ghcjs-8.10.5. There are some errors in `makePackages.sh`.

**Comment by dfordivam** [comment]
I agree that this should be uncommented. I have hit a scenario where the ghc's `./configure` failed and the `makePackages.sh` continued compiling and about 15 min later failed. I had to do the compile again, capture logs and go through logs to figure out where things went wrong.
